.github/workflows/trailing_commas.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+"""This is a prettify script so that flutter formatting can work efficiently.
+
+Methodology:
```

Comment:
You have copied the docstring without reading it. Please update.

Reply:
I can assure you that didn't happen. I will update it with the required Python Docs format